040-SDK/104-delete.mdx

```diff
@@ -127,7 +127,24 @@ users, _ := client.Transaction(context.TODO(), xata.TransactionRequest{
 
 </TabbedCode>
 
-[Transactions](/docs/sdk/transaction) will not fail a delete operation if no record is found. All operations in a transaction must succeed otherwise it is rolled back.
+To delete multiple records we recommend using transactions. [Transactions](/docs/sdk/transaction#delete) will not fail a delete operation if a record is not found. All operations in a transaction must succeed otherwise it is rolled back.
+
+To avoid the SQL proxy's 60-second timeout, delete in small batches. We recommend deleting no more than 10,000 records per batch. You can use a nested `SELECT` statement to manage batch sizes efficiently and prevent timeouts. For example, as in the following:
```

Comment:
```suggestion
Alternatively you can use the `DELETE` SQL statement via the [SQL Proxy](/docs/sdk/sql/overview), but note that there is an implicit timeout of 60 seconds which may be reached when processing many records. To avoid reaching the SQL proxy's timeout, limit the delete statement to smaller batches. You can use a nested `SELECT` statement to control the batch size. For example, as in the following:
```